src/test/rustdoc/pub-use-extern-macros.rs

```diff
@@ -15,6 +15,7 @@
 extern crate macros;
 
 // @has pub_use_extern_macros/macro.bar.html
+// @!has pub_use_extern_macros/index.html 'pub use macros::bar;'
```

Comment:
This needs to be something like `// @!has pub_use_extern_macros/index.html '//code' 'pub use macros::bar;'` to actually test this. The other `@!has` tests in this file need to be fixed in the same way.

Reply:
Oh, that's a good point! It looks like the 2-operator `@has` command doesn't do any HTML parsing. I didn't realize that. `>_>`